packages/docs/pages/accordion-panel.tsx

```diff
@@ -0,0 +1,122 @@
+import { AccordionGroup, H1, Panel, Text } from '@bigcommerce/big-design';
+import React, { useState } from 'react';
+
+import { Code, CodePreview, ContentRoutingTabs, GuidelinesTable, List } from '../components';
+import {
+  AccordionGroupPropTable,
+  AccordionPanelPropTable,
+} from '../PropTables/AccordionPanelPropTable';
+
+const AccordionPanelPage = () => {
+  return (
+    <>
+      <H1>Accordion Panel</H1>
+
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Accordion panels</Code> are containers for content of relative importance to
+          the user that can be selectively expanded or collapsed. They can be useful for reducing
+          the content on a page &amp; the cognitive load for the user.
+        </Text>
+        <Text>
+          An <Code primary>accordion panel</Code> can display different types of content such as
+          text, images, tables, media, buttons, etc. Components are added into the drop zone and
+          render when the user expands the corresponding panel.
+        </Text>
+
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>On similar or redundant information that must be presented together</List.Item>
+          <List.Item>Step-by-step or sequential processes</List.Item>
+        </List>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <CodePreview>
+          {/* jsx-to-string:start */}
+          {function Example() {
+            const [isExpanded, setIsExpanded] = useState(false);
+
+            const onClick = () => {
+              setIsExpanded(!isExpanded);
+            };
```

Comment:
What is this for, when we already manage state inside the component? If this was actually working (by passing these props through the component), we'd see both panels open and close at the same time.